test/io/postgrest.py

```diff
@@ -19,17 +19,17 @@
 
 def sleep_until_postgrest_scache_reload():
     "Sleep until schema cache reload"
-    time.sleep(0.2)
+    time.sleep(0.3)
 
 
 def sleep_until_postgrest_config_reload():
     "Sleep until config reload"
-    time.sleep(0.1)
+    time.sleep(0.2)
 
 
 def sleep_until_postgrest_full_reload():
     "Sleep until schema cache plus config reload"
-    time.sleep(0.2)
+    time.sleep(0.3)
```

Comment:
It looks like the extra query increases a bit the reloading time for the schema cache. Adding the sleep time made the io-tests pass. I'm approving since it doesn't look like a lot. @steve-chavez WDYT?

Reply:
@laurenceisla Seems fine for now.

Although I've been noticing each change to the in-db config query does make the io-tests fail. I wonder if we need to join these queries into one and make them more efficient. 

Anyway, that could be left for later.

Reply:
> It looks like the extra query increases a bit the reloading time for the schema cache.

Maybe it's not correct to rerun the in-db config query for the schema cache too.